app/javascript/components/student/TracksList/List/Track.jsx

```diff
@@ -4,29 +4,28 @@ import { ProgressBar } from './Track/ProgressBar'
 
 export function Track({ track }) {
   return (
-    <tr>
-      <td>
-        <TrackIcon track={track} />
-      </td>
-      <td>{track.title}</td>
-      <td>{track.numConceptExercises}</td>
-      <td>{track.numPracticeExercises}</td>
-      <td>{track.webUrl}</td>
-      <td>{track.isNew.toString()}</td>
-      <td>{track.tags.join(', ')}</td>
-      <td>{track.isJoined.toString()}</td>
-      <td>{track.numCompletedConceptExercises}</td>
-      <td>{track.numCompletedPracticeExercises}</td>
-      <td>
-        {track.isJoined && (
-          <ProgressBar
-            numConceptExercises={track.numConceptExercises}
-            numPracticeExercises={track.numPracticeExercises}
-            numCompletedConceptExercises={track.numCompletedConceptExercises}
-            numCompletedPracticeExercises={track.numCompletedPracticeExercises}
-          />
-        )}
-      </td>
-    </tr>
+    <a className="c-track" href={track.webUrl}>
+      <TrackIcon track={track} />
+      <div className="info">
+        <h3 className="title">{track.title}</h3>
+        {track.isJoined && <div className="joined">Joined</div>}
+        <ul className="counts">
+          <li>
+            {track.numCompletedConceptExercises}/{track.numConceptExercises}{' '}
+            concepts
+          </li>
+          <li>
+            {track.numCompletedPracticeExercises}/{track.numPracticeExercises}{' '}
+            exercises
+          </li>
+        </ul>
+        <ul className="tags">
+          {track.tags.slice(0, 3).map((tag) => {
```

Comment:
We probably want to have some sort of ordering of the tags at some point, with the most important tags being displayed first and such.

Reply:
Yes, good shout. This will actually display the tags in the order they're written to the db, which is probably going to be the order written in the tracks' `config.json` files, so maybe making the order in there meaningful is the best way?

Reply:
Maybe, I don't know. I feel like it might be easy for maintainers to re-order things there. We could perhaps make this explicit in the track's config.json with a separate field?

Reply:
I've created an issue where we can work on this: https://github.com/exercism/v3-project-management/issues/40